### PR TITLE
Upgrade terraform to 0.14 for aws

### PR DESCRIPTION
**Pull request by tei-k**
# Description

https://scalar-labs.atlassian.net/browse/DLT-8330

### CODE REVIEW

File: aws/ca/.terraform.lock.hcl [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# This file is maintained automatically by "terraform init".
```
Comment (feeblefakie) [inline aws/ca/.terraform.lock.hcl:1]: Shouldn't be committed?

File: aws/cassandra/.terraform.lock.hcl [hunk 2]
```diff
@@ -0,0 +1,77 @@
+# This file is maintained automatically by "terraform init".
```
Comment (feeblefakie) [inline aws/cassandra/.terraform.lock.hcl:1]: Ditto.

**Review by feeblefakie — COMMENTED** [review]
Some lock files are committed but not sure if they need to be. Please confirm!

File: aws/ca/.terraform.lock.hcl (hunk 1 above)
Comment (tei-k) [reply aws/ca/.terraform.lock.hcl:1]: https://github.com/scalar-labs/scalar-terraform-examples/pull/13#discussion_r585217817

**Review by ymorimo — APPROVED** [review]
LGTM!

**Review by feeblefakie — APPROVED** [review]
LGTM!

**Pull request merged**